src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -760,4 +820,160 @@
     <value>SSH Host Profile Generator</value>
     <comment>The display name of a dynamic profile generator for SSH hosts</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="DismissSelectionLocalized" xml:space="preserve">

```

Comment:
nit: for these should we call them `xxxxxActionArgumentLocalized` just to make clear what  context they're used in?